src/routes/group/Group.module.css

```diff
@@ -0,0 +1,44 @@
+.container {
+    margin: 2rem 0 0 0;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 2rem 4rem;
+}
+
+@media (max-width : 60em) {
+    .container {
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+
+@media (max-width : 30em) {
+    .container {
+        grid-template-columns: 1fr;
+    }
+}
+
+.title {
+    font-weight: 500;
+    font-size: var(--fs-mid-small-text);
+    color: #551A8B;
```

Comment:
I think the purple color might be misleading as users may believe they can click the title. I suggest using a blue color, bolder and larger font size for both title and description

Reply:
<img width="1001" alt="image" src="https://github.com/Code4UHub/client/assets/60989884/3e038b2a-1ea0-4b66-bc17-d15acedb7fe2">

I think that the color blue is better given the color palette, but I don't think that increasing the size of title and description is the best option (as shown above). I increased the title size, as shown below.

<img width="1496" alt="image" src="https://github.com/Code4UHub/client/assets/60989884/5131e1d6-a2a4-4e98-891b-6ca31d8c6054">